packages/forms-web-app/src/lib/full-appeal/map-planning-application.js

```diff
@@ -0,0 +1,14 @@
+const { APPEAL_ID } = require('@pins/business-rules/src/constants');
```

Comment:
We don't seem to have unit tests for these.

Reply:
This has been added.